src/index.css

```diff
@@ -1,4 +1,15 @@
 @import "./styles/normalize.css";
 
 @import "./styles/header/header.css";
-@import "./styles/content/description.css"
\ No newline at end of file
+@import "./styles/content/description.css";
+@import "./styles/content/digits.css";
+@import "./styles/content/feynman.css";
+@import "./styles/content/kaufman.css";
+@import "./styles/footer/footer.css";
+@import "./styles/shared.css";
```

Comment:
Shared should be imported right after the normilise.css at the beginning of the file because you should be able to rewrite it with the rest stiles but not in reverse.

Reply:
done.

Reply:
Text, check if you pushed the changes before you mark it as done (I had this issue before and I created a personal checklist with "Double check if all the PR changes requests are satisfied")